### ISSUE TITLE
Record the terminal

### ISSUE THREAD

**Issue by felipecrs** [mixn/carbon-now-cli#62]
It would be awesome if this could support taking a screenshot from the terminal text, just like Asciinema, but static.

**Comment by mixn** [comment]
Hey @felipecrs, thanks for opening. 🙂

Could you elaborate a bit on how exactly this would behave / look like?

It sounds like it’s beyond the scope of this project — but I’m still curious!

**Comment by felipecrs** [comment]
For sure!

> It sounds like it’s beyond the scope of this project

Yes, I also thought that but decided to open the Issue anyway so we at least can discuss the topic.

My use case is: I wrote a CLI, and I want to capture some beautiful images using it. Example:

![image](https://user-images.githubusercontent.com/29582865/93775061-d1d21280-fbd6-11ea-9989-6b0e553457c0.png)

I took the screenshot in my Windows Terminal, with my settings and color profiles, but I would like to take more generic screenshots, with the edges properly handled and less of my own environment (there is even a Daft Punk logo in the back, lol).
